pkg/fakerp/routes.go

```diff
@@ -27,4 +27,5 @@ func (s *Server) SetupRoutes() {
 	s.router.Put(filepath.Join("/admin", s.basePath, "/reimage/{hostname}"), s.handleReimage)
 	s.router.Put(filepath.Join("/admin", s.basePath, "/backup/{backupName}"), s.handleBackup)
 	s.router.Put(filepath.Join("/admin", s.basePath, "/runCommand/{hostname}/{command}"), s.handleRunCommand)
+	s.router.Get(filepath.Join("/admin", s.basePath, "/pluginVersion"), s.handleGetPluginVersion)
```

Comment:
`/version` ? And reuse the same structure for all version we will return in the future? instead of building multiple version apis?

Reply:
sure - right now this is a fake API which is not replicated by MSFT so we can do what we like at this stage.